interfaces/matlab_experimental/Setup/buildToolbox.m

```diff
@@ -0,0 +1,104 @@
+function buildToolbox()
+    % Get Cantera version from environment variable
+    version = getenv('TOOLBOX_VERSION');
+    if isempty(version)
+        version = '0.0.0';  % fallback default
+    end
+    fprintf('Building toolbox version: %s\n', version);
+
+    % Read the UUID for the toolbox
+    uuidFile = fullfile(pwd, 'interfaces', 'matlab_experimental', 'Setup', ...
+                        'Cantera_MATLAB_Toolbox.uuid');
+    if isfile(uuidFile)
+        guid = strtrim(fileread(uuidFile));
+        fprintf('The unique identifier for the toolbox is: %s\n', guid);
+    else
+        error('A unique identifier for the toolbox does not exist!');
+    end
+
+    % Define output file
+    outputFile = fullfile(pwd, 'interfaces', 'matlab_experimental', ...
+                          ['Cantera_MATLAB_Toolbox_', version, '.mltbx']);
+
+    % Create temporary folder for reorganizing and faster execution
+    tmpDir = fullfile(pwd, 'tmpDir');
```

Comment:
SCons is configured to create a temporary folder inside the `build` directory. I'd suggest using that folder rather than creating a new one.